server/utils/nunjucksSetup.ts

```diff
@@ -58,4 +58,41 @@ export default function nunjucksSetup(app: express.Express): void {
   njkEnv.addFilter('formatDate', formatDate)
   njkEnv.addGlobal('DPS_HOME_PAGE_URL', config.serviceUrls.digitalPrison)
   njkEnv.addFilter('pluralise', (word, count, plural = `${word}s`) => (count === 1 ? word : plural))
+
+  njkEnv.addFilter('createContactsListRows', contactList => {
+    const activeContactsRows: Array<unknown> = []
+    contactList.forEach((item: Record<string, Date>) => {
+      let addressLine = ''
+      addressLine += item.flat ? `${item.flat}<br />` : ''
```

Comment:
Would prefer we didn't embed HTML <br/> tags into code, or inline functions like this as nunjucks filters because its hard to test them.  It also mixes code with the presentation layer, which is a generally bad idea. I think there are better ways of doing this using SCSS and an address macro that we can re-use elsewhere? I can help with this.

Reply:
Code refactored using addressToLines from hmpps prisoner profile